Queueomatic/Client/Components/Login/LoginComponent.razor

```diff
@@ -0,0 +1,46 @@
+﻿@using Blazored.SessionStorage
+@inject HttpClient _httpClient;
+@inject NavigationManager _navigationManager;
+@inject ISessionStorageService _sessionStorageService;
+
+<LogoComponent/>
+<AuthorizeView>
+    <Authorized>
+        <RedirectHome/>
+    </Authorized>
+</AuthorizeView>
+<div class="component">
+    <div class="form-container">
+        <p class="title">Login</p>
+        <EditForm class="form" Model="_loginDto" OnValidSubmit="Login">
+            <div class="input-group">
+                <label for="username">Email</label>
```

Comment:
This label is incorrect. It should be pointing to the id of the input field

Reply:
Should it? We don't have an id. We use userName which is userEmail as id.

Reply:
Yes it should. This is how forms work in HTML